client/packages/programs/src/JsonForms/common/components/Array/Notes.tsx

```diff
@@ -155,6 +155,10 @@ const NotesComponent = (props: ArrayControlCustomProps) => {
       {...props}
       isElementEditable={isElementEditable}
       getItemLabel={getItemLabel}
+      checkIsError={(child: any) => {
+        if (!child) return false;
+        return child.invalid && !child?.text;
+      }}
```

Comment:
A new general prop for "Array" components -- the ability to pass in a method to check for error state.